docs/sources/release-notes/release-notes-7-3-5.md

```diff
@@ -8,8 +8,15 @@ list = false
 
 # Release notes for Grafana 7.3.5
 
+### Features and enhancements
+
+* **Alerting**: Improve Prometheus Alert Rule error message. [#29390](https://github.com/grafana/grafana/pull/29390), [@wbrowne](https://github.com/wbrowne)
+
 ### Bug fixes
 
+* **Alerting**: Fix alarm message formatting in Dingding. [#29482](https://github.com/grafana/grafana/pull/29482), [@tomowang](https://github.com/tomowang)
 * **AzureMonitor**: Fix unit translation for MilliSeconds. [#29399](https://github.com/grafana/grafana/pull/29399), [@secustor](https://github.com/secustor)
 * **Instrumentation**: Fix bug with invalid handler label value for HTTP request metrics. [#29529](https://github.com/grafana/grafana/pull/29529), [@bergquist](https://github.com/bergquist)
+* **Prometheus**: Fixes problem where changing display name in Field tab had no effect. [#29441](https://github.com/grafana/grafana/pull/29441), [@zoltanbedi](https://github.com/zoltanbedi)
```

Comment:
Please use imperative to reduce variations in grammar:
Use "Fix" on both lines 20 and 21.

Reply:
Good catch @osg-grafana 👍  In that case, should it also be _"..name in Field tab **has** no effect."_?